internal/controller/ansibleRun/ansibleRun.go

```diff
@@ -91,7 +91,7 @@ type ansibleRunner interface {
 }
 
 // Setup adds a controller that reconciles AnsibleRun managed resources.
-func Setup(mgr ctrl.Manager, o controller.Options, ansibleCollectionsPath, ansibleRolesPath string, timeout time.Duration) error {
+func Setup(mgr ctrl.Manager, o controller.Options, ansibleCollectionsPath, ansibleRolesPath string, timeout time.Duration, artifactsHistoryLimit int) error {
```

Comment:
Lmk if you'd like me to refactor so that `Setup`'s list of arguments doesn't keep growing forever

Reply:
yes please ^^

Reply:
Done! Since the config options for the `ansiblerun` controller setup and `config` controller setup keep growing apart, I decided to have a config/settings struct just for `ansiblerunner` (therefore the two `Setup` functions no longer have the same signature. I think it makes sense but lmk if you'd like to do something different

Reply:
thanks